src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApplicationEntity.kt

```diff
@@ -460,7 +460,7 @@ class ApprovedPremisesApplicationEntity(
       else -> {}
     }
 
-    if (noticeType == Cas1ApplicationTimelinessCategory.emergency || noticeType == Cas1ApplicationTimelinessCategory.shortNotice) {
+    if (noticeType == Cas1ApplicationTimelinessCategory.EMERGENCY || noticeType == Cas1ApplicationTimelinessCategory.SHORT_NOTICE) {
```

Comment:
this is going to be an issue because we've used API types in the persistence model, meaning we'll have lowercase values in the database that will no longer resolve properly after this change. I think we probably need to 'fix' this by using an internally managed enumerated type. Let's discuss

Reply:
there's more of these that are less obvious because nothing has changed in the class e.g. Booking.status uses an API enum, i'm sure more do. These will all need fixing (sad face)